templates/prometheus.yaml

```diff
@@ -105,11 +105,6 @@ spec:
                 targetPort: 5601
                 interval: 30s
         prometheusSpec:
-          image:
-            tag: v2.9.2
-          thanos:
```

Comment:
We need the thanos block to run the sidecar

Reply:
Thanks! I restored the `version`, but left the `baseImage` removed since it's the same value as the default: https://github.com/coreos/prometheus-operator/blob/8d44e0990230144177f97cf62ae4f43b1c4e3168/pkg/prometheus/statefulset_test.go#L39